pyproject.toml

```diff
@@ -7,7 +7,7 @@ build-backend = "setuptools.build_meta"
 
 [tool.black]
 target-version = ["py38", "py39", "py310"]
-color = true
+# color = true # Problem with vscode autoformat see: https://github.com/microsoft/vscode-python/issues/16341
```

Comment:
It appears that color is only used for `--diff`. Can you delete this line and add `--color` to `.github/workflows/style.yaml`?

Reply:
> Still not convinced that this is something we want to maintain (why stop at codespaces when there are dozens of other editors and environments with dotfiles we could add?) but figured I would comment on some of the issues I see.

`.devcontainer` are not exclusive to Codespace.
It work with many local IDE (VS-Code, PyCharm ...).
And many cloud service, Codespaces, GitPod